### PR TITLE
extend template for light theme

### PR DESCRIPTION
**Pull request by nicolasmartinrojo**

### CODE REVIEW

File: src/styles/base.js [hunk 1]
```diff
@@ -1114,6 +1115,7 @@ export const theme = {
   selectFontWeight,
   selectColorLT,
   selectFontWeightBold,
+  panelTabColor,
```
Comment (heshan0131) [inline src/styles/base.js:1118]: you need to also add `panelTabColor` to  `themeLT` and `themeBS` and have it equals to the value of `subtextColor` in that theme

File: src/styles/base.js [hunk 2]
```diff
@@ -1310,6 +1312,7 @@ export const theme = {
   lineHeight,
   textColor,
   textColorLT,
+  dataTabletextColor,
```
Comment (heshan0131) [inline src/styles/base.js:1315]: same as above

File: src/styles/base.js [hunk 3]
```diff
@@ -1350,6 +1353,7 @@ export const theme = {
   sliderHandleWidth,
   sliderHandleColor,
   sliderHandleTextColor,
+  sliderInactiveBorderColor,
```
Comment (heshan0131) [inline src/styles/base.js:1356]: same as above

File: src/styles/base.js (hunk 1 above)
Comment (nicolasmartinrojo) [reply src/styles/base.js:1118]: Nice catch!. as both themes extends the theme variable I've added to themeLT and themeBS just the cases where that value was overwritten

File: src/styles/base.js (hunk 2 above)
Comment (nicolasmartinrojo) [reply src/styles/base.js:1315]: sliderHandleTextColor

File: src/styles/base.js (hunk 3 above)
Comment (nicolasmartinrojo) [reply src/styles/base.js:1356]: same as above

File: src/styles/base.js [hunk 4]
```diff
@@ -42,6 +42,7 @@ export const labelColorLT = '#6A7485';
 
 export const textColor = '#A0A7B4';
 export const textColorLT = '#3A414C';
+export const dataTabletextColor = textColorLT;
```
Comment (heshan0131) [inline src/styles/base.js:45]: `dataTableTextColor`

File: src/styles/base.js [hunk 5]
```diff
@@ -1494,6 +1498,7 @@ export const themeLT = {
   sliderBarColor: '#A0A7B4',
   sliderBarBgd: '#D3D8E0',
   sliderHandleColor: '#F7F7F7',
```
Comment (heshan0131) [inline src/styles/base.js:1500]: You created 4 new theme names
```
dataTabletextColor
sliderInactiveBorderColor
panelTabColor
dataTabletextColor
```

Stll missing some in `temeLT` and `themeBS`
Be diligent please

Comment (nicolasmartinrojo) [reply src/styles/base.js:1500]: Both themeLT and ThemeBS extends from `theme`. the variables has been added there(in theme). should I add them again in both themes?

Comment (heshan0131) [reply src/styles/base.js:1500]: for the ones share same value between dark and light, you don't need to, but for the ones have different values you need to add them to light specifically

Comment (nicolasmartinrojo) [reply src/styles/base.js:1500]: This change was attended

**Review by heshan0131 — APPROVED** [review]

**Pull request merged**